css-flexbox/css-flexbox_profile/css/styles.css

```diff
@@ -30,6 +30,16 @@ body {
   color: var(--white);
 }
 
+.profile {
+  display:flex;
+  flex-direction: column;
+  align-items: center;
```

Comment:
if you remove `align-items: center;` and add `align-self:center;` to `.profile__image`, you shouldn't need a style for `.h2`